models/articles.model.js

```diff
@@ -47,23 +47,49 @@ exports.updateArticle = (article_id, newVote) => {
 
 )}
 
-exports.selectArticles = () => {
+exports.selectArticles = (sort_by='created_at', order='desc', topic) => {
+    const valid_sortBy = ['created_at'];
+    const valid_sortOrder = ['asc', 'desc'];
+    const valid_filterTopics = [ 'cats', 'mitch', 'paper'];
```

Comment:
Which topics are valid depends on what's in the topics table. Hard coding these is specific to your test data and won't work in production. 

Query the database to find these out instead 👍